server/controllers/apply/applicationsController.test.ts

```diff
@@ -135,6 +136,7 @@ describe('applicationsController', () => {
         expect(response.render).toHaveBeenCalledWith('applications/confirm', {
           pageHeading: `Confirm ${person.name}'s details`,
           ...person,
+          date: DateFormats.dateObjtoUIDate(new Date()),
           dateOfBirth: DateFormats.isoDateToUIDate(person.dateOfBirth),
```

Comment:
Just a point about this ` dateOfBirth: DateFormats.isoDateToUIDate(person.dateOfBirth),` which I realise isn't part of this PR. Probably:

> Thursday, 20 December 1982

is not how we want to express a DoB? The day of the week is probably a distraction? 

and something like:

> 20 December 1982

or

> 20-Dec-1982

would be clearer and more useful?

Reply:
Yeah, that's a good point. We default to that format as it's the GDS standard, but it makes less sense for a date of birth

Reply:
I've gone with dd/mm/yyyy, which mirrors what is in NOMIS